setup.py

```diff
@@ -33,20 +33,20 @@ def get_version():
 
 # Environment-specific dependencies.
 extras = {
-    "atari": ["ale-py~=0.8.0"],
+    "atari": ["shimmy[atari]>=0.1.0,<1.0"],
     "accept-rom-license": ["autorom[accept-rom-license]~=0.4.2"],
     "box2d": ["box2d-py==2.3.5", "pygame==2.1.0", "swig==4.*"],
     "classic_control": ["pygame==2.1.0"],
     "mujoco_py": ["mujoco_py<2.2,>=2.1"],
-    "mujoco": ["mujoco==2.2", "imageio>=2.14.1"],
+    "mujoco": ["mujoco==2.3", "imageio>=2.14.1"],
```

Comment:
This is a quiet change, why?

Reply:
I can remove, I think I put this in the wrong repo. It will be necessary for python 3.11